src/site/models/universityeventattendees.php

```diff
@@ -55,4 +55,21 @@ public function getListQuery() {
 		return $query;
 	}
 
+	public function getUniversityName() {
```

Comment:
So, this would result in another query!
Take a look at the SwaModelList (which this model extends) it currently has a getMember method. So right now that already joins against the university_member table but will also need to join against the university table to get the university name.
Then the universityname can be accessed in the view by doing $this->getMember()->university or something similar!